src/pickWithDefault.js

```diff
@@ -0,0 +1,36 @@
+var _curry3 = require('./internal/_curry3');
+
+
+/**
+ * Similar to `pick` except that this one includes a `key: <default>` pair for
+ * properties that don't exist
+ *
+ * **Note:** A property set to undefined will be left untouched.
+ *
+ * @func
+ * @memberOf R
+ * @since v0.22.2
+ * @category Object
+ * @sig [k] -> * -> {k: v} -> {k: v}
```

Comment:
I'd like to see:

```haskell
pickWithDefault :: Array String -> a -> StrMap a -> StrMap a
```

We need to start using [`StrMap`][1] in signatures where appropriate if we're to express constraints in type signatures such as this one.


[1]: https://github.com/sanctuary-js/sanctuary-def#strmap